tests/network/sriov.go

```diff
@@ -625,6 +673,16 @@ func getInterfaceNetworkNameByMAC(vmi *v1.VirtualMachineInstance, macAddress str
 	return ""
 }
 
+func getGuestInterfaceNameByMAC(vmi *v1.VirtualMachineInstance, macAddress string) string {
```

Comment:
aint it `getInterfaceNameByMAC` ?

Reply:
The naming of these things are pretty lousy IMO, but 
`getInterfaceNameByMAC` returns the network name as it is on the vmi.spec.networks[0]Name
`getGuestInterfaceNameByMAC` returns the interface name as it looks inside the vmi guest (eth0, eth1, ...)
Although the both could be considered as returning the `InterfaceName` they return completely different things, so I don't see a reason to combine the 2 functions into 1.

Reply:
```
func getInterfaceNameByMAC(vmi *v1.VirtualMachineInstance, mac string) (string, error) {
	for _, iface := range vmi.Status.Interfaces {
		if iface.MAC == mac {
			return iface.InterfaceName, nil
		}
	}
	return "", fmt.Errorf("could not get sriov interface by MAC: no interface on VMI %s with MAC %s", vmi.Name, mac)
}
```

It returns `InterfaceName` as well (it is not `getInterfaceNetworkNameByMAC`)
isn't it ?

Reply:
ah, I thought you were talking about a different function `getInterfaceNetworkNameByMAC` .. yes you are totally right!! thanks :)